tasks/install.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Download latest dokuwiki
+  get_url:
+    url: https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz
+    dest: /tmp/dokuwiki-stable.tgz
+ 
+- name: Unarchive dokuwiki-stable.tgz to {{ dokuwiki_install_path }}
+  unarchive:
+    src: /tmp/dokuwiki-stable.tgz
+    dest: "{{ dokuwiki_install_path | dirname }}"
+    owner: www-data
+    remote_src: yes
+    list_files: yes
+  register: file_list
+
+- name: Version name from folder path
+  file:
+    src: "{{ dokuwiki_install_path | dirname }}/{{ file_list['files'][0] }}"
+    dest: "{{ dokuwiki_install_path }}"
+    owner: www-data
+    state: link
+
+- name: Install xml-php package used by dokuwiki.
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - php-ldap
```

Comment:
According to the title, only one package is being installed.
Also, `php-ldap` is only needed if the user configures `dokuwiki_authtype`=`ldap`, right?

Reply:
done